apps/api/src/persistence/repositories/arango-id-repository.ts

```diff
@@ -64,6 +64,7 @@ export class ArangoIdRepository implements IIdRepository {
     async create(id: AggregateId): Promise<void> {
         const databaseDocument = mapUuidDocumentToDatabaseDocument({
             id,
+            // we need to be consistent with dates
```

Comment:
Add a story for this and do some research and get this right.

Reply:
[Here](https://www.pivotaltracker.com/story/show/186660975) is a story for this.